tests/integration/test_sqlalchemy_integration.py

```diff
@@ -421,10 +428,10 @@ def test_get_view_names(trino_connection, schema):
         )
         metadata.create_all(engine)
         view_name = schema_name + ".test_get_view_names"
-        conn.execute(f"CREATE VIEW {view_name} AS SELECT * FROM test_table")
+        conn.execute(sqla.text(f"CREATE VIEW {view_name} AS SELECT * FROM test_table"))
```

Comment:
This `conn` object is a dbapi connection (see fixture `trino_connection`) so why do we need this sqla.text here?

Reply:
Ah, 🤦  - should've seen what class the fixture is coming from. Thanks.